README.md

```diff
@@ -245,7 +245,7 @@ List of currently supported languages:
 - [x] [json](https://github.com/tree-sitter/tree-sitter-json) (maintained by @steelsojka)
 - [ ] [julia](https://github.com/tree-sitter/tree-sitter-julia)
 - [x] [lua](https://github.com/nvim-treesitter/tree-sitter-lua) (maintained by @vigoux)
-- [ ] [nix](https://github.com/cstrahan/tree-sitter-nix)
+- [x] [nix](https://github.com/cstrahan/tree-sitter-nix)
 - [x] [ocaml](https://github.com/tree-sitter/tree-sitter-ocaml) (maintained by @undu)
```

Comment:
Are you aware that this part of the readme is autogenerated. You have to add yourself as the maintainer in parsers.lua

Reply:
No, thank you.